autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/ui/credential/management/AutofillSettingsViewModel.kt

```diff
@@ -169,7 +172,8 @@ class AutofillSettingsViewModel @Inject constructor(
                 // if credential mode started with edit, it means that we need to exit credential mode right away instead of going
                 // back to view mode.
                 if (it is EditingExisting && !it.startedCredentialModeWithEdit) {
-                    _viewState.value = value.copy(credentialMode = Viewing(credentialsViewed = it.credentialsViewed))
+                    _viewState.value =
+                        value.copy(credentialMode = Viewing(credentialsViewed = it.credentialsViewed, it.credentialsViewed.shouldShowLinkButton()))
```

Comment:
NIT: I would add the named argument `showLinkButton=` for consistency and to avoid future conflicts in case the order of the arguments changes